modules/concepts/services/_index.md

```diff
@@ -150,6 +150,21 @@ To do it: you declare your new service using the old service name. So if you wan
 
 That's done. The service registered under the name `prestashop.core.b2b.b2b_feature` is now your service. The previous `prestashop.core.b2b.b2b_feature` is gone.
 
+Since {{< minver v=8.1 >}}, some services definitions have been updated to rely on FQCN. Nothing changes, just name your service after the original FQCN :
```

Comment:
```suggestion
Since {{< minver v=8.1 >}}, some service definitions have been updated to rely on FQCN. Nothing changes, just name your service after the original FQCN:
```